content/docs/cucumber/api.md

```diff
@@ -848,6 +848,31 @@ The **Command-Line Interface Runner (CLI Runner)** is an executable Java class t
 ```
 java io.cucumber.core.cli.Main
 ```
+Note that you will need to add the following jars to your classpath, in addition to the location of your compiled .class files:
+
+* `cucumber-core`
+* `gherkin`
+* `cucumber-expressions`
+* `datatable`
+* `datatable-dependencies`
+* `cucumber-java`
```

Comment:
`cucumber-java` is not strictly necessary. It could also be `cucumber-java8`.

And with the release of v5 this list is also no longer accurate. It added a `docstring`, `cucumber-gherkin` and `cucumber-gherkin-vintage` modules. :smile: 

So is it absolutely necessary to include this list? Would it be possible to say make sure you have `cucumber-core` and its transitive dependencies? Maybe we can refer to to the java skeleton project?

Reply:
Yes, that simplifies things a lot and makes the docs easier to manage.  I removed the list of jars and added the bit about transitive dependencies.  Can you re-review please?  Thanks for looking over this! :)